static/js/place/chart_block.tsx

```diff
@@ -117,7 +166,15 @@ class ChartBlock extends React.Component<ChartBlockPropType> {
           dcid={this.props.dcid}
           chartType={chartTypeEnum.LINE}
           trend={this.props.data.trend}
-          title={`${this.props.data.title} in ${this.displayPlaceName}`}
+          title={intl.formatMessage(
+            {
+              id: "chart_clause:variable_in_place",
+              defaultMessage: "{variable} in {place}",
+              description:
+                "Used for chart titles like Unemployment rate in Texas or Poverty rate in California.",
```

Comment:
we might have to describe this a certain way. in the meantime, could you make it clear in the description where variable and place is used?

Reply:
Maybe: "Used for chart titles about a variable for a place, e.g. "{Unemployment rate} in {USA}" or "{Poverty rate} in {California}", where variable is the statistical variable of the chart, and place is the name of a place.

Side note: s/place/place_name/ for the name variable.

Reply:
I updated to placeName. Happy to change all the variable place holders to snake case if that's better.